core/src/test/java/org/apache/iceberg/TestRemoveSnapshots.java

```diff
@@ -1455,6 +1455,56 @@ public void testMaxSnapshotAgeMultipleBranches() {
     Assert.assertNull(table.ops().current().snapshot(initialSnapshotId));
   }
 
+  @Test
+  public void testRetainFilesOnRetainedBranches() {
+    // Append a file to main and test branch
+    String testBranch = "test-branch";
+    table.newAppend().appendFile(FILE_A).commit();
+    Snapshot a = table.currentSnapshot();
+    table.manageSnapshots().createBranch(testBranch, a.snapshotId()).commit();
+
+    // Delete A from main
+    table.newDelete().deleteFile(FILE_A).commit();
+    Snapshot deletionA = table.currentSnapshot();
+    // Add B to main
+    table.newAppend().appendFile(FILE_B).commit();
+    long tAfterCommits = waitUntilAfter(table.currentSnapshot().timestampMillis());
+    Set<String> deletedFiles = Sets.newHashSet();
```

Comment:
I don't think these declarations have much to do with the block of code "Add B to main" so they should be separated by an empty line.